### PR TITLE
[Update] Adding a method to create an entity naturally

### PR DESCRIPTION
**Pull request by rexbut**
API | [Common](https://github.com/SpongePowered/SpongeCommon/pull/923)

To fix conflicts, I recreated a new branch.

### CODE REVIEW

File: src/main/java/org/spongepowered/api/world/extent/EntityUniverse.java [hunk 1]
```diff
@@ -127,6 +127,44 @@
...
+     * systems.</p>
+     *
+     * @param type The type
+     * @param position The position
+     * @return An entity, if one was created
+     * @throws IllegalArgumentException If the position or entity type is not valid to create
+     * @throws IllegalStateException If a constructor cannot be found
+     */
+    default Entity createEntityNaturally(EntityType type, Vector3i position) throws IllegalArgumentException, IllegalStateException {
+        return createEntity(type, position.toDouble());
```
Comment (kashike) [inline src/main/java/org/spongepowered/api/world/extent/EntityUniverse.java]: I assume you meant to call `createEntityNaturally` here.

Comment (rexbut) [reply src/main/java/org/spongepowered/api/world/extent/EntityUniverse.java]: Thank you for the report

File: src/main/java/org/spongepowered/api/world/extent/EntityUniverse.java [hunk 2]
```diff
@@ -127,6 +127,44 @@
...
+     * systems.</p>
+     *
+     * @param type The type
+     * @param position The position
+     * @return An entity, if one was created
+     * @throws IllegalArgumentException If the position or entity type is not valid to create
+     * @throws IllegalStateException If a constructor cannot be found
+     */
+    default Entity createEntityNaturally(EntityType type, Vector3i position) throws IllegalArgumentException, IllegalStateException {
+        return createEntityNaturally(type, position.toDouble());
```
Comment (ST-DDT) [inline src/main/java/org/spongepowered/api/world/extent/EntityUniverse.java:168]: `position` should be `checkNotNull`ed first. The same goes for the `createEntity` version.

Comment (rexbut) [reply src/main/java/org/spongepowered/api/world/extent/EntityUniverse.java:168]: Okay

**Comment by Aaron1011** [comment]
Sorry for the long delay in getting this merged.

**Pull request merged**